src/assets/scripts/client/trafficGenerator/SpawnPatternCollection.js

```diff
@@ -111,6 +127,38 @@ export default class SpawnPatternCollection extends BaseCollection {
     }
 
     /**
+     * Gather a randomized list of departure `SpawnPatternModel` objects
+     * that will be created during the prSpawn.
+     *
+     * Clamps max at 5 patterns.
+     * Randomly selects departure patterns from a filtered list containing only departure patterns.
+     *
+     * This method should be called by the `SpawnScheduler` when preSpawning new
+     * aircraft on session start.
+     *
+     * @for SpawnPatternCollection
+     * @method getDepartureModelsForPreSpawn
+     * @return departureModelsForPreSpawn {array<SpawnPatternModel>}
+     */
+    getDepartureModelsForPreSpawn() {
+        const departureModelsForPreSpawn = [];
+        const minimumDeparturesToPreSpawn = 1;
+        const maximumDeparturesToPreSpawn = 5;
```

Comment:
yeah, 10 minutes gives us 6 departures.

fun fact: `_.sum(_.map())` line above could also be done with `_reduce()` (which is also a native javascript function, though not widely supported). 
```javascript
const departuresPerHour = _reduce(this.departureModels, (sum, spawnPattern) => sum + spawnPattern.rate, 0);
```